app/javascript/components/visualization/controls/DifferentialExpressionGroupPicker.jsx

```diff
@@ -36,37 +36,55 @@ function parseDeFile(tsvText, isAuthorDe=false) {
     if (tsvLine === '') {continue}
     const row = tsvLine.split('\t')
 
-    // Each element in this array is DE data for the gene in this row
-    //
-    // TODO: There are usually more columns than size (e.g. logfoldchanges)
-    // and significance (e.g. pvals_adj) that may well be of interest.
-    // However, we don't parse those here before there is no UI for them.
-    // If we opt to build a UI for further metrics (e.g. pctNzGroup in Scanpy
-    // / pct.1 in Seurat, etc.), we would need to order them canonically
-    // across SCP-computed DE results and (Ingest Pipeline-processed) author
-    // DE results.
-    let [
-      index, // eslint-disable-line
-      name, size, significance
-    ] = row
-
+    let deGene
 
     if (isAuthorDe) {
-      size = round(size, 3),
-      significance = round(significance, 3)
-    }
+      // Each element in this array is DE data for the gene in this row
+      //
+      // TODO: There are usually more columns than size (e.g. logfoldchanges)
+      // and significance (e.g. pvals_adj) that may well be of interest.
+      // However, we don't parse those here before there is no UI for them.
+      // If we opt to build a UI for further metrics (e.g. pctNzGroup in Scanpy
+      // / pct.1 in Seurat, etc.), we would need to order them canonically
+      // across SCP-computed DE results and (Ingest Pipeline-processed) author
+      // DE results.
+      let [
+        index, // eslint-disable-line
+        name, size, significance
+      ] = row
+
+      if (isAuthorDe) {
+        size = round(size, 3),
+        significance = round(significance, 3)
+      }
 
-    const deGene = {
-      // TODO (SCP-5201): Show significant zeros, e.g. 0's to right of 9 in 0.900
-      size,
-      significance
+      deGene = {
+        // TODO (SCP-5201): Show significant zeros, e.g. 0's to right of 9 in 0.900
+        name,
+        size,
+        significance
+      }
+    } else {
+      // names  scores  logfoldchanges  pvals pvals_adj pct_nz_group  pct_nz_reference
+      const [
+        index, // eslint-disable-line
+        name, score, size, altSignificance,	significance
```

Comment:
It looks like there's a stray tab character here.

Reply:
Thanks, tidied in 36a7cc6.